tests/unit_and_functional/test_k8s_service_deployments.py

```diff
@@ -471,17 +472,82 @@ def test_list_service_stage_deployments_returns_service_stage_info(
                     )
                 )
                 deployment_info = list_service_stage_deployments(service_namespace)
+                mock_k8s_apps_api().list_namespaced_deployment.assert_called_with(
+                    namespace=service_namespace, label_selector="app=bodywork"
+                )
                 assert service_name in deployment_info.keys()
                 assert deployment_info[service_name]["service_url"] == service_url
                 assert deployment_info[service_name]["service_port"] == service_port
-                assert deployment_info[service_name]["service_exposed"] == "true"
+                assert deployment_info[service_name]["service_exposed"] is True
                 assert deployment_info[service_name]["available_replicas"] == 1
                 assert deployment_info[service_name]["unavailable_replicas"] == 0
                 assert (
                     deployment_info[service_name]["git_branch"] == "project_repo_branch"
                 )  # noqa
                 assert deployment_info[service_name]["git_url"] == "project_repo_url"
-                assert deployment_info[service_name]["has_ingress"] == "true"
+                assert deployment_info[service_name]["has_ingress"] is True
+
+
+@patch("kubernetes.client.AppsV1Api")
+def test_list_service_stage_deployments_returns_all_services_on_cluster(
+    mock_k8s_apps_api: MagicMock,
+    service_stage_deployment_object: kubernetes.client.V1Deployment,
+):
+    service_namespace = service_stage_deployment_object.metadata.namespace
+    service_name = service_stage_deployment_object.metadata.name
+
+    service_stage_deployment_object.status = kubernetes.client.V1DeploymentStatus(
```

Comment:
An anally retentive voice in my head is telling me that this should be named `service_stage_deployment_object1`...

Reply:
Changed to silence the voice 🙂

Reply:
Reverting this change, it's because that's the name of the fixture that returns the object up top therefore it would be weird to rename this with the 1 at the end (and in all the other tests that use it).